FWCore/ParameterSetReader/bin/edmConfigHash.cpp

```diff
@@ -29,7 +29,7 @@ int main(int argc, char **argv) try {
     config = std::string(argv[1]);
   }
 
-  std::shared_ptr<edm::ParameterSet> parameterSet = edm::boost_python::readConfig(config);
+  std::shared_ptr<edm::ParameterSet> parameterSet = edm::readConfig(config);
```

Comment:
ok, pybind11 is already inside ParameterSetReader, so no need to call it explicitly...